src/handleTask.js

```diff
@@ -0,0 +1,59 @@
+export const tasks = localStorage.getItem('tasks') ? JSON.parse(localStorage.getItem('tasks')) : [];
+
+export default class Task {
+  constructor(description, index) {
+    this.description = description;
+    this.completed = false;
+    this.index = index;
+  }
+}
+
+export const addTask = (description) => {
+  const task = new Task(description, tasks.length + 1);
+  const newTasks = [...tasks, task];
+  localStorage.setItem('tasks', JSON.stringify(newTasks));
+  document.location.reload();
+};
+
+const removeTask = (index) => {
+  const newTasks = tasks.filter((task) => task.index !== index);
+  newTasks.map((tasks, index) => {
+    tasks.index = index + 1;
+    return tasks;
+  });
+  localStorage.setItem('tasks', JSON.stringify(newTasks));
+  document.location.reload();
```

Comment:
- [ ] Please refrain from using `document.location.reload( )`. Too many reloads of the page is considered bad UX and makes the page slow and laggy (especially if you get data from a back-end API). Instead, you can remove the specific node from the DOM by first selecting it using querySelector and then removing it using `document.removeChild()`. Even adding and editing can be done in similar ways. Please remove all the document.location.reload() commands from your Application.

Reply:
Thank you so far, i get the point and it could be better to implement it in the upcoming work, kindly i suggest that this comment be **OPTIONNAL** 🙏🙏🙏🙏